lti_store/models.py

```diff
@@ -59,16 +62,40 @@ class ExternalLtiConfiguration(models.Model):
 
     # LTI 1.3 Related variables
     lti_1p3_client_id = models.CharField(
+        "LTI 1.3 Client ID",
         max_length=255,
         blank=True,
         help_text=_("Client ID used by LTI tool"),
     )
+    lti_1p3_deployment_id = models.CharField(
+        "LTI 1.3 Deployment ID",
+        max_length=255,
+        blank=True,
+        help_text=_("Deployment ID used by LTI tool"),
+    )
+    lti_1p3_oidc_url = models.CharField(
+        "LTI 1.3 OIDC URL",
+        max_length=255,
+        blank=True,
+        help_text="This is the OIDC third-party initiated login endpoint URL in the LTI 1.3 flow, "
+                  "which should be provided by the LTI Tool."
+    )
+    lti_1p3_launch_url = models.CharField(
+        "LTI 1.3 Launch URL",
+        max_length=255,
+        blank=True,
+        help_text="This is the LTI launch URL, otherwise known as the target_link_uri. "
```

Comment:
```suggestion
        help_text="This is the LTI launch URL, otherwise known as the target_link_uri."
```
Please, use multi-line comments, @kuipumu